timemachine/ff/handlers/nonbonded.py

```diff
@@ -391,6 +415,22 @@ def static_parameterize(mol):
 
 
 class AM1CCCHandler(SerializableMixIn):
+    """The AM1CCCHandler stands for AM1 Correctable Charge Correction (CCC) which uses OpenEye's AM1 charges[1]
+    along with corrections provided by the Forcefield definition in the form of smirks and charge deltas.
+
+    This handler supports back propagation with respect to ligand parameters, which is what the "Correctable" refers
```

Comment:
nit: wording -- differentiation w.r.t. ligand-specific system parameters is supported regardless of handler definition. (Want to emphasize difference between ligand-specific system parameters and forcefield-wide shared parameters.)

```suggestion
    This handler supports back propagation with respect to the charge deltas, which is what the "Correctable" refers
```

Reply:
Oh! Already raised in https://github.com/proteneer/timemachine/pull/677#discussion_r831371002 and addressed in https://github.com/proteneer/timemachine/pull/677/commits/98055a8f4c8e7e8f1453cfa2f5b561cf6e6fea10